tests/grpcio-wheel/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, poetry2nix, python3, pkgs, runCommand }:
+let
+  inherit (pkgs.stdenv) isLinux;
+  env = poetry2nix.mkPoetryEnv {
+    python = python3;
+    pyproject = ./pyproject.toml;
+    poetrylock = ./poetry.lock;
+    preferWheels = false;
+    overrides = poetry2nix.overrides.withDefaults (
+      self: super: {
+        grpcio = super.grpcio.override {
+          preferWheel = isLinux;
+        };
+      }
+    );
+  };
+  isWheelGrpcIO = env.python.pkgs.grpcio.src.isWheel;
+in
+assert isLinux -> isWheelGrpcIO; runCommand "grpcio-wheel" { } ''
+  ${env}/bin/python -c 'import grpc; print(grpc.__version__)' > $out
```

Comment:
What's the advantage of evaluating to this rathern than just `env`?

Reply:
Nothing I guess. I wanted to test the import, but `env` is probably sufficient.